modules/cython/3.0.11-1/MODULE.bazel

```diff
@@ -0,0 +1,25 @@
+module(
+    name = "cython",
+    version = "3.0.11",
```

Comment:
Should be `3.0.11-1`?

Reply:
Good catch. Would be nice if `//tools:bcr_validation` would have caught this as well.

Reply:
Done